Index: linyaps-bak/libs/oci-cfg-generators/src/linglong/oci-cfg-generators/container_cfg_builder.cpp
===================================================================
--- linyaps-bak.orig/libs/oci-cfg-generators/src/linglong/oci-cfg-generators/container_cfg_builder.cpp
+++ linyaps-bak/libs/oci-cfg-generators/src/linglong/oci-cfg-generators/container_cfg_builder.cpp
@@ -586,6 +591,8 @@ ContainerCfgBuilder &ContainerCfgBuilder
        bindIfExist(*hostStaticsMount, loc);
     }
 
+    bindIfExist(*hostStaticsMount, "/persistent", "/persistent", false);
+
     return *this;
 }
 
