diff --git a/libs/linglong/src/linglong/runtime/run_context.cpp b/libs/linglong/src/linglong/runtime/run_context.cpp
--- a/libs/linglong/src/linglong/runtime/run_context.cpp
+++ b/libs/linglong/src/linglong/runtime/run_context.cpp
@@ -847,8 +847,14 @@ void RunContext::resolveHostDynamic()
 void RunContext::resolveHostDynamic()
 {
     LINGLONG_TRACE("resolve host dynamic paths");
 
-    constexpr std::array<const char *, 0> paths{};
+    constexpr std::array<const char *, 5> paths{
+        "/usr/share/wallpapers",
+        "/usr/share/music",
+        "/usr/share/extensions-from-app-store",
+        "/usr/share/dsg",
+        "/usr/share/deepin/distribution.info",
+    };
 
     std::vector<api::types::v1::Mount> hostDynamic;
     hostDynamic.reserve(paths.size());
